src/qt/locale/bitcoin_ko_KR.ts

```diff
@@ -1771,13 +1771,13 @@ p, li { white-space: pre-wrap; }
         <source>BIP70 payment requests are deprecated and disabled by default. Restart with -enable-bip70 if you absolutely have to use this functionality.
 
 Use this functionality with extreme caution.</source>
-        <translation type="unfinished"></translation>
+        <translation>해당 기능 사용시 매우 주의하기 바랍니다</translation>
```

Comment:
For me, `the sentence` in the source above doesn't look translated into `translation`.